src/ngraph/runtime/gcpu/unit_test.manifest

```diff
@@ -7,6 +7,11 @@ convert_bf16_float32
 GCPU.onnx_model_quant_conv_linear
 GCPU.onnx_top_k_opset_10
 
```

Comment:
GCPU should work. It also calls evaluate.

Reply:
GCPU doesn't work, I already tried because of the result tensor.